cmsdb/processes/azh.py

```diff
@@ -0,0 +1,1860 @@
+    "azh_ttll_a2100_h1900",
+    "azh_ttll_a2100_h2000",
+    "azh_ttll_a2100_h350",
+    "azh_ttll_a2100_h400",
+    "azh_ttll_a2100_h500",
+    "azh_ttll_a2100_h600",
+    "azh_ttll_a2100_h700",
+    "azh_ttll_a2100_h800",
+    "azh_ttll_a2100_h900",
+    "azh_ttll_a430_h330",
+    "azh_ttll_a450_h330",
+    "azh_ttll_a450_h350",
+    "azh_ttll_a500_h330",
+    "azh_ttll_a500_h350",
+    "azh_ttll_a500_h370",
+    "azh_ttll_a500_h400",
+    "azh_ttll_a550_h330",
+    "azh_ttll_a550_h350",
+    "azh_ttll_a550_h400",
+    "azh_ttll_a550_h450",
+    "azh_ttll_a600_h330",
+    "azh_ttll_a600_h350",
+    "azh_ttll_a600_h400",
+    "azh_ttll_a600_h450",
+    "azh_ttll_a600_h500",
+    "azh_ttll_a650_h330",
+    "azh_ttll_a650_h350",
+    "azh_ttll_a650_h400",
+    "azh_ttll_a650_h450",
+    "azh_ttll_a650_h500",
+    "azh_ttll_a650_h550",
+    "azh_ttll_a700_h330",
+    "azh_ttll_a700_h350",
+    "azh_ttll_a700_h370",
+    "azh_ttll_a700_h400",
+    "azh_ttll_a700_h450",
+    "azh_ttll_a700_h500",
+    "azh_ttll_a700_h550",
+    "azh_ttll_a750_h330",
+    "azh_ttll_a750_h350",
+    "azh_ttll_a750_h400",
+    "azh_ttll_a750_h450",
+    "azh_ttll_a750_h500",
+    "azh_ttll_a750_h550",
+    "azh_ttll_a750_h600",
+    "azh_ttll_a750_h650",
+    "azh_ttll_a800_h330",
+    "azh_ttll_a800_h350",
+    "azh_ttll_a800_h400",
+    "azh_ttll_a800_h450",
+    "azh_ttll_a800_h500",
+    "azh_ttll_a800_h550",
+    "azh_ttll_a800_h600",
+    "azh_ttll_a800_h650",
+    "azh_ttll_a800_h700",
+    "azh_ttll_a850_h330",
+    "azh_ttll_a850_h350",
+    "azh_ttll_a850_h400",
+    "azh_ttll_a850_h450",
+    "azh_ttll_a850_h500",
+    "azh_ttll_a850_h550",
+    "azh_ttll_a850_h600",
+    "azh_ttll_a850_h650",
+    "azh_ttll_a850_h700",
+    "azh_ttll_a850_h750",
+    "azh_ttll_a900_h330",
+    "azh_ttll_a900_h350",
+    "azh_ttll_a900_h370",
+    "azh_ttll_a900_h400",
+    "azh_ttll_a900_h450",
+    "azh_ttll_a900_h550",
+    "azh_ttll_a900_h500",
+    "azh_ttll_a900_h600",
+    "azh_ttll_a900_h650",
+    "azh_ttll_a900_h700",
+    "azh_ttll_a900_h750",
+    "azh_ttll_a900_h800",
+    "azh_ttll_a950_h330",
+    "azh_ttll_a950_h350",
+    "azh_ttll_a950_h400",
+    "azh_ttll_a950_h450",
+    "azh_ttll_a950_h500",
+    "azh_ttll_a950_h550",
+    "azh_ttll_a950_h600",
+    "azh_ttll_a950_h650",
+    "azh_ttll_a950_h700",
+    "azh_ttll_a950_h750",
+    "azh_ttll_a950_h800",
+    "azh_ttll_a950_h850",
+]
+
+
+from order import Process
+from scinum import Number
+
+#
+# A->ZH
+#
+
+azh = Process(
+    name="azh",
+    label=r"AtoZH",
+    id=1000000,
+    xsecs={13: Number(0.1)},  # TODO
+)
+
+
+azh_ttll_a1000_h330 = azh.add_process(
+ name="azh_ttll_a1000_h330",
+ label=rf"{azh.label} ($m_A = 1000$ GeV, $m_h = 330$ GeV)",
+ id=1000001,
+ xsecs={13: Number(0.1)}, # TODO
+ )
+azh_ttll_a1000_h350 = azh.add_process(
+ name="azh_ttll_a1000_h350",
+ label=rf"{azh.label} ($m_A = 1000$ GeV, $m_h = 350$ GeV)",
+ id=1000002,
+ xsecs={13: Number(0.1)}, # TODO
+ )
```

Comment:
This is where the linter is giving errors. I think the following would be correct:
```suggestion
azh_ttll_a1000_h350 = azh.add_process(
     name="azh_ttll_a1000_h350",
     label=rf"{azh.label} ($m_A = 1000$ GeV, $m_h = 350$ GeV)",
     id=1000002,
     xsecs={13: Number(0.1)}, #  TODO
 )
```